datadog/Kubecostdashboard.json

```diff
@@ -0,0 +1,774 @@
+{
+  "title": "Kubecost dashboard",
+  "description": "",
+  "widgets": [
+    {
+      "id": 2344982212878190,
+      "definition": {
+        "title": "Cluster Costs Overview",
+        "background_color": "vivid_green",
+        "show_title": true,
+        "type": "group",
+        "layout_type": "ordered",
+        "widgets": [
+          {
+            "id": 6129707067870730,
+            "definition": {
+              "type": "image",
+              "url": "https://kubecost.awsworkshop.io/images/kubecost_logo.png",
+              "sizing": "contain",
+              "margin": "md",
+              "has_background": false,
+              "has_border": false,
+              "vertical_align": "center",
+              "horizontal_align": "center"
+            },
+            "layout": { "x": 0, "y": 0, "width": 12, "height": 1 }
+          },
+          {
+            "id": 6087428645000872,
+            "definition": {
+              "title": "Total Monthly Cost",
```

Comment:
what do you think of adding (onDemand) to all titles?
I know is a little messy but it sets expectations.

Reply:
@jessegoodier Let me know how do you think https://p.us5.datadoghq.com/sb/7043b676-87a8-11ed-9cfc-da7ad0900005-4a3de4f7ad2574d0174ee669265f95d6

Reply:
Updated in the latest commit